client_internals_it_test.go

```diff
@@ -488,6 +490,71 @@ func proxyManagerShutdownTest(t *testing.T) {
 	require.True(t, ok)
 }
 
+func infiniteRestartTest(t *testing.T) {
```

Comment:
infinite reconnect?

Reply:
OK, changed the test name at: https://github.com/hazelcast/hazelcast-go-client/pull/927/commits/1a676da1b5abfe84fad771dbe3bf54f2b9d7d56f